ffi/ffi_wrapper.go

```diff
@@ -1,13 +1,14 @@
 package main
 
-/*
-#include <stdlib.h>
-*/
+// #include <stdio.h>
```

Comment:
Do these comments have a special meaning (or should they be removed?

Reply:
the do - they instruct the cgo to what files to incude